.github/ISSUE_TEMPLATE/1.bug_report.yml

```
@@ -0,0 +1,205 @@
+
+        If you need help with your own project, you can start a discussion in the ["Q&A" section](https://github.com/public-ui/kolibri/discussions/new?category=q-a)
+
+        #### Deutsch:
+        Diese Vorlage ist zur Meldung von KoliBri-Bugs. Dokumentationsbezogene Bugs sollten stattdessen mit dieser [Vorlage](https://github.com/public-ui/kolibri/issues/new/choose) gemeldet werden.
+        Funktionsanfragen sollten als [Diskussionen](https://github.com/public-ui/kolibri/discussions/new/choose) eröffnet werden.
+
+        Bevor Sie ein neues Problem melden, führen Sie bitte eine [Suche](https://github.com/public-ui/kolibri/issues) nach bestehenden Problemen durch und stimmen Sie dem bestehenden Problem mit :+1: zu. Dies führt zu einer schnelleren Lösung.
+
+        Wenn Sie Hilfe bei Ihrem eigenen Projekt benötigen, können Sie eine Diskussion im ["Q&A"-Bereich](https://github.com/public-ui/kolibri/discussions/new?category=q-a) starten.
+
+        Please fill out the form: / Bitte fülle die folgende Form aus:
+        ------------------------------------------------------------------------------------------------------
+  - type: checkboxes
+    id: verify_release
+    attributes:
+      label: Verify current release / Überprüfung der aktuellen Version
+      options:
+        - label: I verified that the issue exists in the latest KoliBri v1 or v2 release / Ich habe mich vergewissert, dass das Problem in der neuesten Version von KoliBri v1 oder v2 auftritt
+          required: true
+  - type: input
+    id: reproduction_url
+    attributes:
+      label: Link to the code that reproduces this issue / Link zu dem Code, der dieses Problem reproduziert
+      description: |
+        A link to a **public** GitHub repository or a StackBlitz minimal reproduction. Minimal reproductions should be created from our actual released versions (e.g. v1, v2, ...) and should include only changes that contribute to the issue. StackBlitz templates can be found under the followjng url's:
+
+        #### Deutsch:
+        Ein Link zu einem **öffentlichen** GitHub-Repository oder einer StackBlitz-Minimalreproduktion. Minimale Reproduktionen sollten aus unseren aktuell veröffentlichten Versionen (z. B. v1, v2, ...) erstellt werden und nur Änderungen enthalten, die zum Problem beitragen. StackBlitz-Vorlagen können unter den folgenden URLs gefunden werden:
+
+        - [KoliBri (v1)](https://stackblitz.com/edit/vitejs-vite-dcg6xo)
+        - [KoliBri (v2)](https://stackblitz.com/edit/vitejs-vite-kkfhk5)
+      placeholder: 'https://github.com/user/my-minimal-kolibri-issue-reproduction or https://stackblitz.com/edit/XXXXXX'
+    validations:
+      required: false
+  - type: dropdown
+    id: affected_themes
+    attributes:
+      label: Which theme(s) are affected? / Welche(s) Layout(s) ist/sind betroffen?
```

```suggestion
      label: Which theme(s) are affected? / Welche(s) Theme(s) ist/sind betroffen?
```
(Wir haben es auch auf dt. immer als Themes bezeichnet)